rclcpp/include/rclcpp/client.hpp

```diff
@@ -581,7 +640,7 @@ class Client : public ClientBase
     PromiseWithRequest promise;
     auto shared_future = promise.get_future().share();
     auto req_id = async_send_request_impl(
-      *request,
+      request,
```

Comment:
std::move missing here?

Reply:
`request` is also used in line 646, if I move it, would be invalid